awscli/botocore/utils.py

```diff
@@ -970,7 +970,7 @@ def _generate_skeleton(self, shape, stack, name=''):
                 if self._use_member_names:
```

Comment:
Let's make sure to add change log for this. You can run the `scripts/new-change` script to add one. I would recommend labeling it as an enhancement for `--generate-cli-skeleton`.